Feature/ffmt 1914

sneha-nerekar opened the pull request:
Return receipt changes - Added Unit price and Return Reduction Amount column, changed the column positions

kibo-snehanerekar committed:
FFMT-1914 - Return receipt changes

kibo-snehanerekar committed:
FFMT-1914 - Changed total label to Reduction Amount

sagarraut0007 commented on labels/en-US.json:
```diff
@@ -553,7 +553,9 @@
-  "returnReceiptItemTotalColumn": "Total",
+  "returnReceiptItemTotalColumn": "Item Total",
+  "returnReceiptReductionAmountColumn": "Return Reduction Amount",
+  "returnReceiptUnitPriceColumn": "Unit Price",
```
seeing some discrepancies in labels files like "returnReceiptUnitPriceColumn" key which you have added into en-US.json is not present in de-DE.josn. Make sure your newly added keys are present in both files. 
Note: Keys should be sorted alphabetically. (You already took care of it but in case if you missed any)

shuhao-kibo replied:
ditto

sneha-nerekar replied:
Done

sagarraut0007 commented on templates/back-office/return-receipt-content.hypr:
```diff
@@ -1,60 +1,73 @@
-        </td>
-    </tr>
-</table>	
+<style>
```
Can you move this inline style to respective .less/css files?

sagarraut0007 reviewed (changes requested):
requested minor changes. please take a look

shuhao-kibo commented on templates/back-office/return-receipt-content.hypr:
```diff
@@ -1,60 +1,73 @@
-    </tr>
-</table>	
+<style>
+  pre {
```
what is for? I'm not seeing any `pre` contents

sneha-nerekar replied:
Removed. Earlier added to test the JSON

shuhao-kibo commented on templates/back-office/return-receipt-content.hypr:
```diff
@@ -1,60 +1,73 @@
+            <th>{{ labels.returnReceiptItemUPCColumn }}</th>
+            <th>{{ labels.returnReceiptItemNameColumn }}</th>
+            <th>{{ labels.returnReceiptItemStatusColumn }}</th>
+            <th class="align-center">{{ labels.returnReceiptUnitPriceColumn }}</th>
+            <th class="align-center">{{ labels.returnReceiptItemTotalColumn }}</th>
```
why do these two columns need to align center but not the other columns?

sneha-nerekar replied:
amounts/numbers look good and aligned when aligned centered -
For Example - 
![image](https://user-images.githubusercontent.com/67549927/102317180-1e8e7500-3f9d-11eb-9f4a-94dc3792651b.png)

shuhao-kibo commented on labels/de-DE.json:
```diff
@@ -31,6 +31,7 @@
   "applyCredit": "Bewerben Kredit",
   "availableBalance": "Verfügbares Guthaben",
   "availableNow": "Jetzt erhältlich",
+  "backorderEst": "Geschätztes Wiederauffüllungsdatum",
```
what's this for?

sneha-nerekar replied:
Actually, this came from the merged commit. It is not used for this ticket

shuhao-kibo replied:
Can you remove it if it's not used in this ticket?

sneha-nerekar replied:
done

shuhao-kibo commented on labels/de-DE.json:
```diff
@@ -136,6 +137,7 @@
   "customerAtCurbsideBlob4": "Notez s'il vous plaît...<br/><ul><li>Veuillez apporter votre pièce d'identité qui correspond au nom sur votre commande (sauf indication contraire).</li><li>Si votre colis n'est pas récupéré dans les 5 jours ouvrables, votre commande sera annulée et vous serez remboursé.</li><li>Lorsque vous arrivez au magasin, veuillez vous enregistrer via le lien fourni dans le proc ...
   "customerAtCurbsideLetUsKnow": "LAISSEZ-NOUS SAVOIR LIVRER VOS ARTICLES!",
   "customerAtCurbsideWithYouShortly": "NOUS SERONS AVEC VOUS Bientôt!",
+  "customerCare": "Kundendienst",
```
and this one is for?

sneha-nerekar replied:
This as well came from the merged commit. It is not used for this ticket

kibo-snehanerekar committed:
FFMT-1914 - Review comment changes

Also removed Total word as discussed with Tripti

kibo-snehanerekar committed:
FFMT-1914 - Code review change

Removed unused labels from de-DE.json

kibo-snehanerekar committed:
FFMT-1914 - changed label

As discussed with Tripti changed the label for Return Reduction to Refund Amount

kibo-snehanerekar committed:
FFMT-1914 - Resolved item total scenario
